### PR TITLE
Included the travis ci for the coreclr

### PR DESCRIPTION
**Pull request by naveensrinivasan**
Included the travis CI for the coreclr builds.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Octokit - GitHub API Client Library for .NET [![Build Status](https://ci.appveyor.com/api/projects/status/github/octokit/octokit.net?branch=master)](https://ci.appveyor.com/project/Haacked15676/octokit-net)
+# Octokit - GitHub API Client Library for .NET [![Build Status](https://ci.appveyor.com/api/projects/status/github/octokit/octokit.net?branch=master)](https://ci.appveyor.com/project/Haacked15676/octokit-net) [![Build Status](https://travis-ci.org/naveensrinivasan/octokit.net.svg)](https://travis-ci.org/naveensrinivasan/octokit.net)
```
Comment (shiftkey) [inline README.md]: This should be the final URL - https://travis-ci.org/octokit/octokit.net - could you update this badge?

Comment (naveensrinivasan) [inline README.md]: @shiftkey You could update it when you get a travis-ci setup for octokit.net . I won't know the exact URL until one is setup.

Comment (shiftkey) [inline README.md]: Do you see the "No builds for this repository" message at that above URL?

**Comment by shiftkey** [comment]
This markup shows the build's "unknown" badge:

```
[![Build Status](https://travis-ci.org/octokit/octokit.net.svg)](https://travis-ci.org/octokit/octokit.net)
```

i.e.

[Build Status](https://travis-ci.org/octokit/octokit.net)

@naveensrinivasan if you can add a new commit to this branch you should see it trigger (I've enabled it only when the `.travis.yml` file is present...

**Comment by naveensrinivasan** [comment]
@shiftkey  I pushed the changes https://travis-ci.org/octokit/octokit.net/jobs/95660459 and it build successfully. I don't know why it is showing as build unknow.

**Comment by shiftkey** [comment]
@naveensrinivasan I'd say this is because the badge status is looking at the default branch (which hasn't been built yet!)

**Comment by shiftkey** [comment]
Looks like the AppVeyor error is a bit different to what we're tracking in #983

**Comment by shiftkey** [comment]
@naveensrinivasan if you merge `master` into this it should be ready to merge and close out!

**Comment by shiftkey** [comment]
Oh wait, that build error is on me to fix. I'll take this in :metal:

**Comment by naveensrinivasan** [comment]
:joy:

**Pull request merged**